specs/merge/validator.md

```diff
@@ -74,6 +74,14 @@ def get_pow_block_at_total_difficulty(total_difficulty: uint256, pow_chain: Sequ
 
     return None
 
+def get_terminal_pow_block(total_difficulty: uint256, block_hash_override: Hash32, pow_chain: Sequence[PowBlock]) -> Optional[PowBlock]:
```

Comment:
I think we could use `TERMINAL_*` config parameters directly for clarity, the function could look as follows:
```python
def get_terminal_pow_block(pow_chain: Sequence[PowBlock]) -> Optional[PowBlock]:
    # Check block hash override prior to terminal total difficulty
    if TERMINAL_BLOCK_HASH != Hash32():
        pow_block_overrides = [pow_block for pow_block in pow_chain if pow_block.block_hash == TERMINAL_BLOCK_HASH]
        if len(pow_block_overrides) != 0:
            return pow_block_overrides[0]
        else:
            return None

    return get_pow_block_at_total_difficulty(TERMINAL_TOTAL_DIFFICULTY, pow_chain)
```

Reply:
ya agreed, wil change

Reply:
(and nice catch on the slight logic issue)